frontend/src/features/admin-form/create/featureTour/constants.ts

```diff
@@ -0,0 +1,27 @@
+interface FeatureTourStepAttributes {
+  id: string
+  title: string
+  content: string
+}
+
+type FeatureStep = 'FIRST_STEP' | 'SECOND_STEP' | 'THIRD_STEP'
+
+export const FEATURE_TOUR: Record<FeatureStep, FeatureTourStepAttributes> = {
+  FIRST_STEP: {
+    id: 'feature_tour_first_step',
+    title: 'Build your form',
+    content: 'Create forms easily by dragging and dropping fields',
+  },
+  SECOND_STEP: {
+    id: 'feature_tour_second_step',
+    title: 'Design your form',
+    content:
+      'Change your agency logo, theme colours, layout and add Instructions for your page',
+  },
+  THIRD_STEP: {
+    id: 'feature_tour_third_step',
+    title: 'Add Logic to your form',
+    content:
+      "Personalise your user's experience by showing fields and sections, or disabling submissions based on their input",
+  },
+}
```

Comment:
Should we change this to an array instead of using a FeatureStep key? basically `FEATURE_TOUR[0] === FEATURE_TOUR.FIRST_STEP` anyways. 

then re: the above comment, we can reexport a `FEATURE_STEPS` constant using map like so:
```ts
export const FEATURE_STEPS = FEATURE_TOUR.map(({ id, ...rest}) => ({
  ...rest,
  target: `#${id}`,
  disableBeacon: true
}))
```

bonus if we can type this constant to joyride's step type.